subcommands/common_config.go

```diff
@@ -139,7 +139,7 @@ func PrintConfig(cfg *client.DeviceConfig, showAppliedAt, highlightFirstLine boo
 }
 
 type SetUpdatesConfigOptions struct {
-	UpdateTags     string
+	UpdateTag      string
```

Comment:
sorry - i wasn't clear in my previous comment. this is kind of backwards. I was trying to say - make the CLI only accept "updateTag". The backend is still technically "tags". But lets change fioctl to only let people configure a single tag. 

If this is a pain, we can drop this commit and do it as a separate PR?

Reply:
Then I prefer to drop this commit.

A change you propose would be backward incompatible and might break some users expectations.
I'm not a fan to add a new logic and break an old logic within one feature.

It's not so trivial to be able to preserve multi-tag behavior for already present configs, and only disallow it for new configs.